### ISSUE TITLE
Pretty-print in place without changes

### ISSUE THREAD

**Issue by benmoss** [mikefarah/yq#715]
**Please describe your feature request.**
I'd like to be able to take an existing YAML file and pretty-print it in-place without making any edits to it.

```foo.yaml
foo: [1,2,3]
```

Today `eval -P` works:
```
$  yq eval -P foo.yaml
foo:
  - 1
  - 2
  - 3
 ```

but `eval -Pi` errors:
```
$ yq eval -Pi foo.yaml
Error: Write inplace flag only applicable when giving an expression and at least one file
```

**Comment by mikefarah** [comment]
Yep that's a bug in the flag parsing, for now you can:

```
yq e -i '... style=""' examples/data1.yaml
```

**Comment by benmoss** [comment]
Sounds good, wasn't sure if this was the same as #716 but definitely was me trying to work around the same problem :)

**Comment by mikefarah** [comment]
Thanks for the PR! Released in 4.5.1 :)
